delta-api/src/main/java/io/cdap/delta/api/SourceProperties.java

```diff
@@ -26,19 +26,15 @@
   /**
    * Enum describing ordering of events.
    */
-  public enum Ordering {
-    ORDERED,
-    UN_ORDERED
-  }
 
-  private final Ordering ordering;
+  private final boolean supportRowId;
```

Comment:
nit: should we rename this to `rowIdSupported` and corresponding getter can be `isRowIdSupported` and setter `setRowIdSupported`?

Reply:
done :)